content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -15,6 +15,76 @@ This document serves both as a reference to the values and as a coordination poi
 
 ## Labels, annotations and taints used on API objects
 
+### app.kubernetes.io/component
+
+Example: `app.kubernetes.io/component=database`
```

Comment:
The other examples all use `=` but the example on https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels shows the examples as they are in YAML (with `:`).

Is using `=` the correct convention?

Reply:
`:` _is_ correct for YAML, which I think is what we should assume we're writing a snippet of (it's definitely not JSON).

I'd use `=` here and then file another issue about making the switch to YAML format.

Reply:
https://deploy-preview-32991--kubernetes-io-main-staging.netlify.app/docs/reference/labels-annotations-taints/